packages/frontend/src/components/common/Filters/FilterInputs/FilterStringAutoComplete.tsx

```diff
@@ -156,44 +157,48 @@ const FilterStringAutoComplete: FC<Props> = ({
     // memo override component so list doesn't scroll to the top on each click
     const DropdownComponentOverride = useCallback(
         ({ children, ...props }: { children: ReactNode }) => (
-            <ScrollArea {...props}>
-                {refreshedAt && healthData?.hasCacheAutocompleResults ? (
-                    <Tooltip
-                        withinPortal
-                        position="left"
-                        label={`Click here to cache results now`}
-                    >
+            <Stack w="100%" spacing={0}>
```

Comment:
do you need this stack?
will test in Render soon

Reply:
only if this feature is enabled, because it adds a new component to the bottom (without this, it will add one next to the other)